pkgs/applications/editors/emacs/elisp-packages/manual-packages/sunrise-commander/default.nix

```diff
@@ -4,13 +4,13 @@
 , emacs
 }:
 
-trivialBuild rec {
+trivialBuild {
```

Comment:
`melpaBuild` is better because it handles a multi-file package better.  See https://www.gnu.org/software/emacs/manual/html_node/elisp/Multi_002dfile-Packages.html

Reply:
I will move this commit to another branch